From mst@dev.mellanox.co.il Sun Jun 10 12:39:08 2007
From: "Michael S. Tsirkin" <mst@dev.mellanox.co.il>
Date: Sun, 10 Jun 2007 22:39:12 +0300
Subject: Driver core: include linux/mutex.h from attribute_container.c
Cc: James Bottomley <James.Bottomley@steeleye.com>, Andrew Morton <akpm@linux-foundation.org>, gregkh@suse.de
Message-ID: <20070610193912.GA2252@mellanox.co.il>
Content-Disposition: inline


attribute_container.c uses DEFINE_MUTEX, so while
linux/mutex.h seems to be pulled in indirectly
by one of the headers it includes, the right thing
is to include linux/mutex.h directly.

Signed-off-by: Michael S. Tsirkin <mst@dev.mellanox.co.il>

---

---
 drivers/base/attribute_container.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/base/attribute_container.c
+++ b/drivers/base/attribute_container.c
@@ -18,6 +18,7 @@
 #include <linux/slab.h>
 #include <linux/list.h>
 #include <linux/module.h>
+#include <linux/mutex.h>
 
 #include "base.h"
 
